lib/dry/schema/message/or/multi_path.rb

```diff
@@ -34,17 +34,17 @@ def to_h
             end
           end
 
-          # @api private
-          def self.handler(message)
-            handlers.find { |k,| message.is_a?(k) }&.last
-          end
+          MULTI_PATH_HANDLER = -> { _1 }
```

Comment:
We have `IDENTITY` function defined in `Dry::Core::Constants` which is supposed to be available here, please check.